crates/wasmtime/src/runtime/vm/gc/gc_ref.rs

```diff
@@ -88,8 +88,10 @@ impl VMGcHeader {
     pub fn set_reserved_u27(&mut self, value: u32) {
         assert!(
             VMGcKind::value_fits_in_unused_bits(value),
-            "VMGcHeader::set_reserved_u26 with value using more than 26 bits"
+            "VMGcHeader::set_reserved_u27 with value using more than 27 bits: \
+             {value:#034b} ({value}, {value:#010x})"
         );
+        self.kind &= VMGcKind::MASK;
```

Comment:
This was such a silly bug but it made me feel like I was losing my mind :facepalm: